libqtile/command/base.py

```diff
@@ -54,14 +56,102 @@ class CommandException(Exception):
     """Error raised while executing a command"""
 
 
-class CommandObject(metaclass=abc.ABCMeta):
+class _CommandMeta(abc.ABCMeta):
+    _updated = set()
+
+    def __new__(cls, name, bases, dct):
+        # Get the class definition
+        instance = super().__new__(cls, name, bases, dct)
+
+        # If we've already processed this class, we can pass this to __init__ now
+        if name in cls._updated:
+            return instance
```

Comment:
This actually causes problems. Looks like we need to do the parsing of commands for each `instance`.